packages/client/components/TaskWatermark.tsx

```diff
@@ -1,51 +1,47 @@
-import styled from '@emotion/styled'
+import React from 'react'
 import AzureDevOpsSVG from './AzureDevOpsSVG'
 import GitHubSVG from './GitHubSVG'
 import GitLabSVG from './GitLabSVG'
 import JiraSVG from './JiraSVG'
 import JiraServerSVG from './JiraServerSVG'
+import LinearSVG from './LinearSVG'
+
+// We need to update the SVG components to accept size props or create wrapper components
+// For this implementation, we'll create a wrapper that ensures 120x120 sizing
+
+interface WatermarkSVGProps {
+  Icon: React.ComponentType
+}
+
+const WatermarkSVG: React.FC<WatermarkSVGProps> = ({Icon}) => (
+  <div className='absolute right-[24px] bottom-[8px] scale-500 transform'>
+    <Icon />
+  </div>
+)
 
 const iconLookup = {
   _xGitHubIssue: GitHubSVG,
   JiraIssue: JiraSVG,
   JiraServerIssue: JiraServerSVG,
   _xGitLabIssue: GitLabSVG,
-  AzureDevOpsWorkItem: AzureDevOpsSVG
+  AzureDevOpsWorkItem: AzureDevOpsSVG,
+  _xLinearIssue: LinearSVG
 } as const
 
-const WatermarkBlock = styled('div')({
-  bottom: 0,
-  left: 0,
-  overflow: 'hidden',
-  pointerEvents: 'none',
-  position: 'absolute',
-  right: 0,
-  textAlign: 'center',
-  top: 0,
-  verticalAlign: 'middle',
-  opacity: 0.2,
-  zIndex: 1,
-  svg: {
```

Comment:
+1 I wonder why the `svg` styling is no longer necessary?

Reply:
Oh boy. Long story on this one but suffice to say: I had to update the styles that surround the component everywhere else in order to make it so that this component could be refactored to Tailwind